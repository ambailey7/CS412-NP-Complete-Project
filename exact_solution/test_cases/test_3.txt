11 55
0 1 727.603
0 2 254.979
0 3 919.321
0 4 586.685
0 5 709.178
0 6 876.858
0 7 574.844
0 8 378.652
0 9 182.119
0 10 72.711
1 2 807.196
1 3 344.019
1 4 787.561
1 5 755.857
1 6 123.104
1 7 888.497
1 8 658.266
1 9 666.484
1 10 5.024
2 3 189.58
2 4 798.455
2 5 561.211
2 6 534.718
2 7 67.073
2 8 158.102
2 9 122.298
2 10 632.915
3 4 225.196
3 5 575.142
3 6 90.186
3 7 477.57
3 8 983.659
3 9 422.076
3 10 408.908
4 5 877.752
4 6 232.128
4 7 48.11
4 8 455.412
4 9 77.648
4 10 448.006
5 6 143.825
5 7 302.687
5 8 590.023
5 9 37.026
5 10 390.941
6 7 368.775
6 8 15.407
6 9 766.667
6 10 884.685
7 8 502.034
7 9 545.543
7 10 747.653
8 9 495.925
8 10 790.098
9 10 729.936