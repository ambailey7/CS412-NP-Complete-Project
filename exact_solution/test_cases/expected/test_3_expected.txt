1298.612
8 2 7 4 3 1 10 0 9 5 6 8